src/pages/forside/sections/4-personinfo/3-adresser/visning/adresser/Bostedsadresse.tsx

```diff
@@ -0,0 +1,60 @@
+import React from "react";
+import { Bostedsadresse as IBostedsadresse } from "types/adresser/bostedsadresse";
+import Vegadresse from "../adressetyper/norske-adresser/Vegadresse";
+import UtenlanskAdresse from "../adressetyper/utenlanske-adresser/UtenlanskAdresse";
+import Matrikkeladresse from "../adressetyper/norske-adresser/Matrikkeladresse";
+import Ukjentbosted from "../adressetyper/norske-adresser/Ukjentbosted";
+import AdressePanel from "../../komponenter/AdressePanel";
+import moment from "moment";
+import "moment/locale/nb";
+
+moment.locale("nb");
+
+interface Props {
+  bostedsadresse: IBostedsadresse;
+}
+
+const Bostedsadresse = (props: Props) => {
+  const flyttedato = props.bostedsadresse.angittFlyttedato;
+  const flyttedatoFormatert = flyttedato ? moment(flyttedato).format("L") : "";
+
+  let adresse;
+  let kommune;
+  let bruksenhetsnummer;
+
+  switch (props.bostedsadresse?.adresse?.type) {
```

Comment:
// TODO: Lage funkjson for håndtering av adresse som kan benyttes fra Bostedsadresse, DeltBosted, Kontaktadresse og Oppholdsadresse

Reply:
Lagt til // TODO. Tenker det er greit å gjøre dette i en egen runde så vi kan få ut alle branchene som er avhengige av denne.

Reply:
Prøvde meg på en fix likevel, var ikke så mye endringer som skulle til